lambdas/revoke-vcs/src/main/java/uk/gov/di/ipv/core/revokevcs/RevokeVcsHandler.java

```diff
@@ -39,6 +41,8 @@
 @SuppressWarnings("unused") // Temporarily disable to pass sonarqube
 public class RevokeVcsHandler implements RequestStreamHandler {
     private static final Logger LOGGER = LogManager.getLogger();
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+    private final RevokeVcsResult result = new RevokeVcsResult();
```

Comment:
Be very careful using class variables like this - they might persist between invocations of the lambda. It'd be much better to initialise this in the handler method.

Reply:
oh! ok, will do that thank you